test/verify/check-kdump

```diff
@@ -175,6 +175,62 @@ class TestKdump(MachineCase):
         self.assertNotEqual(
             m.execute("""find "{0}" -maxdepth 1 -mindepth 1 -type d -exec echo {{}} \;""".format(customPath)), "")
 
+    @nondestructive
+    def testConfiguration(self):
+        b = self.browser
+        m = self.machine
+
+        self.restore_file("/etc/kdump.conf")
+
+        m.execute("systemctl disable --now kdump || true")
```

Comment:
I think it's ok to not restore the original status after the test. But I don't like the `|| true` here -- is there a reason why stopping/disabling would fail?